lib/engine/game/g_18_norway/game.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+require_relative 'meta'
+require_relative '../base'
+require_relative 'companies'
+require_relative 'corporations'
+require_relative 'map'
+require_relative 'trains'
+require_relative 'phases'
+
+module Engine
+  module Game
+    module G18Norway
+      class Game < Game::Base
+        include_meta(G18Norway::Meta)
+        include Companies
+        include Corporations
+        include Map
+        include Trains
+        include Phases
+
+        register_colors(red: '#d1232a',
+                        orange: '#f58121',
+                        black: '#110a0c',
+                        blue: '#025aaa',
+                        lightBlue: '#8dd7f6',
+                        yellow: '#ffe600',
+                        green: '#32763f',
+                        brightGreen: '#6ec037')
+
+        TRACK_RESTRICTION = :permissive
+        SELL_BUY_ORDER = :sell_buy
+        TILE_RESERVATION_BLOCKS_OTHERS = :always
+        CURRENCY_FORMAT_STR = '%skr'
+        EBUY_SELL_MORE_THAN_NEEDED = true
+        CAPITALIZATION = :incremental
+        MUST_BUY_TRAIN = :always
+        POOL_SHARE_DROP = :left_block
+        SELL_AFTER = :p_any_operate
+        SELL_MOVEMENT = :left_block
+        CERT_LIMIT_COUNTS_BANKRUPTED = true
+        HOME_TOKEN_TIMING = :float
+
+        BANK_CASH = 999_000
+
+        CERT_LIMIT = {
+          3 => { 0 => 12, 1 => 12, 2 => 12, 3 => 15, 4 => 15, 5 => 17, 6 => 17, 7 => 19, 8 => 19 },
+          4 => { 0 => 9, 1 => 9, 2 => 9, 3 => 11, 4 => 11, 5 => 13, 6 => 13, 7 => 15, 8 => 15 },
+          5 => { 0 => 7, 1 => 7, 2 => 7, 3 => 9, 4 => 9, 5 => 10, 6 => 10, 7 => 12, 8 => 12 },
+        }.freeze
+
+        STARTING_CASH = { 3 => 400, 4 => 300, 5 => 240 }.freeze
+
+        MARKET = [
+          %w[0c 10 20 30 40
+             50p 60p 70p 80p 90p 100p 112p 124p 137p 150p
+             165 180 195 220 245 270 300 330 365 400 440 480],
+           ].freeze
```

Comment:
It would be a good idea to have different coloured zones for the 10–40 and 165–480 ranges, to make it clearer that the stock market movement is different in those areas (only president sales affects share price, and triple jumps possible).

Reply:
18 Norway is almost the same as 1846 when it comes to stock market but there is a difference when it comes to selling of shares. All sales affects the share price and not only the president as in 1846

Reply:
If a company's share price is less than 50 then the share price only moves left if the president sells a share. There's a different zone on the [stock market](https://github.com/akvik/18Norway/blob/main/18Norway_Market.pdf) to show this.

Reply:
Good catch, I had completely missed this

Reply:
Updated